Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.h

```diff
@@ -221,6 +225,8 @@ class WebInspectorUIProxy
     void evaluateInFrontendForTesting(const String&);
 
 private:
+    const RefPtr<WebInspectorBackendProxy> m_backend;
```

Comment:
```suggestion
    const Ref<WebInspectorBackendProxy> m_backend;
```
I think all of these are `Ref` now. Don't see a need to deviate here, but maybe I'm missing something?

Reply:
Makes sense, I was initially trying to resolve the safer-cpp checks with RefPtr but the const part was what I needed.